libkbfs/config_local.go

```diff
@@ -713,28 +713,6 @@ func (c *ConfigLocal) SetLoggerMaker(
 	c.loggerFn = loggerFn
 }
 
-// newConfigLocalWithCryptoForSigning initializes a local crypto
-// config w/a crypto interface, using the given signing key, that can
-// be used for non-PKI crypto.
-func newConfigLocalWithCryptoForSigning(
-	signingKey kbfscrypto.SigningKey) *ConfigLocal {
-	config := NewConfigLocal()
-	config.SetLoggerMaker(func(m string) logger.Logger {
-		return logger.NewNull()
-	})
-	cryptPrivateKey := MakeLocalUserCryptPrivateKeyOrBust("nobody")
-	crypto := NewCryptoLocal(config.Codec(), signingKey, cryptPrivateKey)
-	config.SetCrypto(crypto)
-	return config
-}
-
-// NewConfigLocalWithCrypto initializes a local crypto config w/a
-// crypto interface that can be used for non-PKI crypto.
-func NewConfigLocalWithCrypto() *ConfigLocal {
```

Comment:
This is used on the server side, but https://github.com/keybase/kbfs-server/pull/179 removes that.